custom/theme.scss

```diff
@@ -1,6 +1,6 @@
-$light-background: #e9e5fb;
```

Comment:
Consider using consistent naming conventions for color variables to maintain a unified style.

For reference, see [Sass Guidelines: Naming Conventions](https://sass-guidelin.es/#naming-conventions).